ui/Rhino/AGS/dev/AGS_check_dof_cmd.py

```diff
@@ -27,11 +27,11 @@ def RunCommand(is_interactive):
     inds = len(list(form.diagram.edges_where({'is_ind': True})))
 
     if k == inds:
-        print('Correct number of loads selected.')
+        print('Correct number of loads selected. The system is statically determined')
```

Comment:
sorry, my comment about "statically determined structures" was probably a bit confusing. i would not include that in the text because it is a property of the system, regardless of how many independent edges the user selects.

let's just keep it simple for now and only say something about the correctness of the number of assigned forces.

* Success: You have identified the correct number of externally applied loads.
* Warning: You have not yet identified all external loads.
* Warning: You have identified too many external forces as loads.

or something like that